projects/igniteui-angular/src/lib/tabs/tabs/tabs.component.ts

```diff
@@ -138,10 +138,11 @@ export class IgxTabsComponent extends IgxTabsDirective implements AfterViewInit,
     constructor(
         @Inject(IgxAngularAnimationService) animationService: AnimationService,
         cdr: ChangeDetectorRef,
+        platform: PlatformUtil,
         private ngZone: NgZone,
         dir: IgxDirectionality,
     ) {
-        super(animationService, cdr, dir);
+        super(animationService, cdr, platform, dir);
```

Comment:
Eh, why push it to the base if it's not needed? You can keep it as prop for this class like the zone, but also you can do the same check as in utils and check `getResizeObserver()` actually resolves. Would make for the smallest change.

Reply:
PS: Not for this fix, but would be nice if all the uses of the resize observer used the one util method.